cicd/forgeops-tests/Dockerfile

```diff
@@ -1,11 +1,13 @@
 # Dockerfile with dependencies for forgeops-tests
 FROM python:3.5-alpine
 
-RUN apk add --no-cache bash
+RUN apk add --no-cache bash gcc musl-dev libffi-dev openssl-dev
```

Comment:
Requirements for kubernetes.
Finding the minimal list of dependencies took a bit of effort.

Reply:
This file can be removed